chapters/hub/build-pod-image.md

```diff
@@ -1,39 +1,45 @@
 # Build Your Pod into a Docker Image
 
 ## Goal
+The objective of this tutorial is to serve as an elaborate guide for Pod image usage.
+Users can use Pod Image via several ways:
 
-Instead of 
-```bash
-jina pod --uses hub/example/mwu_encoder.yml --port-in 55555 --port-out 55556
-```
-
-After this tutorial, you can use the Pod image via:
-```bash
-docker run jinaai/hub.examples.mwu_encoder --port-in 55555 --port-out 55556
-```
-
-...or use the Pod image in the Flow API:
-```python
-from jina.flow import Flow
+- Run with Docker (`docker run`)
+  - ```
+    bash docker run jinaai/hub.examples.mwu_encoder --port-in 55555 --port-out 55556
```

Comment:
no need for bash right?

Reply:
`bash` should be connected to the top of the code fence so we get syntax highlight (if supported)